benchmark/convert_nil.yaml

```diff
@@ -1,9 +1,7 @@
 loop_count: 100
 contexts:
   - gems:
-      csv: 3.0.1
-  - gems:
-      csv: 3.0.2
```

Comment:
We should not remove old versions because we use them as base line.

Reply:
@kou  Done